product_docs/docs/biganimal/release/using_cluster/pgd_cli_ba.mdx

```diff
@@ -0,0 +1,114 @@
+!!!
+
+To use the PGD CLI with your database connection string, use the `--dsn` flag with your PGD CLI command:
+
+```bash
+pgd show-nodes --dsn "<your_connection_string>"
+```
+
+## PGD commands in BigAnimal
+
+!!! Note
+There are three EDB Postgres Distributed CLI commands that don't apply to BigAnimal: `create-proxy`, `delete-proxy` and `alter-proxy-option`.
+!!!
+
+What follows are some examples of the most common PGD CLI commands with a BigAnimal cluster.
+
+### `pgd check-health`
+
+`pgd check-health` provides statuses with relevant messaging regarding the clock skew of node pairs, node accessibility, the current raft leader, replication slot health, and versioning consistency:
+
+```
+$ pgd check-health --dsn "postgres://edb_admin@p-mbx2p83u9n-a.pg.biganimal.io:5432/bdrdb?sslmode=require"
+__OUTPUT__
+Check      Status  Message                                                    
+-----      ------  -------                                                    
+ClockSkew  Ok      All BDR node pairs have clockskew within permissible limit 
+Connection Ok      All BDR nodes are accessible                               
+Raft       Warning There is no RAFT_LEADER, an election might be in progress  
+Replslots  Ok      All BDR replication slots are working correctly            
+Version    Ok      All nodes are running same BDR versions               
+```
+
+### `pgd show-nodes`
+
+`pgd show-nodes` returns all the nodes in the DHA cluster and their summaries, including name, node id, group, and current/target state:
+
+```
+$ pgd show-nodes --dsn "postgres://edb_admin@p-mbx2p83u9n-a.pg.biganimal.io:5432/bdrdb?sslmode=require"
+__OUTPUT__
+Node             Node ID    Group          Type Current State Target State Status Seq ID 
+----             -------    -----          ---- ------------- ------------ ------ ------ 
+p-mbx2p83u9n-a-1 3537039754 dc1            data ACTIVE        ACTIVE       Up     1      
+p-mbx2p83u9n-a-2 3155790619 p-mbx2p83u9n-a data ACTIVE        ACTIVE       Up     2      
+p-mbx2p83u9n-a-3 2604177211 p-mbx2p83u9n-a data ACTIVE        ACTIVE       Up     3      
+```
+
+### `pgd show-groups`
+
+`pgd show-groups` returns all groups in your DHA BigAnimal cluster. It also notes which node is the current write leader of each group:
+
+
+```
+$ pgd show-groups --dsn "postgres://edb_admin@p-mbx2p83u9n-a.pg.biganimal.io:5432/bdrdb?sslmode=require"
+__OUTPUT__
+Group          Group ID   Type   Parent Group   Location Raft  Routing Write Leader     
+-----          --------   ----   ------------   -------- ----  ------- ------------     
+world          3239291720 global                         true  true    p-mbx2p83u9n-a-1 
+dc1            4269540889 data   p-mbx2p83u9n-a          false false                    
+p-mbx2p83u9n-a 2800873689 data   world                   true  true    p-mbx2p83u9n-a-3 
+```
+
+### `pgd switchover`
+
+`pgd switchover` changes the write leader of the group to another node without a failover.
```

Comment:
@jpe442 this wording feels a little odd to me - maybe instead something like allows you to manually change the write leader of the group, and can be used to simulate failover https://www.enterprisedb.com/docs/pgd/latest/quickstart/further_explore_failover/?

Reply:
Okay, I will use your wording.